LayoutTests/imported/w3c/resources/import-expectations.json

```diff
@@ -2,6 +2,7 @@
     "web-platform-tests": "skip-new-directories",
     "web-platform-tests/.github": "skip",
     "web-platform-tests/.well-known": "skip",
+    "web-platform-tests//html/semantics/the-button-element/command-and-commandfor": "import",
```

Comment:
You don't need this since html/ is imported

Reply:
Ah that was just auto added by the import script.

Reply:
Ah I included a leading slash and it must have got confused. Will remove now.